src/main.html

```diff
@@ -1,66 +1,71 @@
-    <link rel="apple-touch-icon" sizes="76x76"
-      href="<style-url>/images/favicons/apple-touch-icon-76x76.png">
-    <link rel="apple-touch-icon" sizes="152x152"
-      href="<style-url>/images/favicons/apple-touch-icon-152x152.png">
-    <link rel="apple-touch-icon" sizes="180x180"
-      href="<style-url>/images/favicons/apple-touch-icon-180x180.png">
-    <link rel="icon" type="image/png" sizes="192x192"
-      href="<style-url>/images/favicons/favicon-192x192.png">
-    <link rel="icon" type="image/png" sizes="160x160"
-      href="<style-url>/images/favicons/favicon-160x160.png">
-    <link rel="icon" type="image/png" sizes="96x96"
-      href="<style-url>/images/favicons/favicon-96x96.png">
-    <link rel="icon" type="image/png" sizes="16x16"
-      href="<style-url>/images/favicons/favicon-16x16.png" >
-    <link rel="icon" type="image/png" sizes="32x32"
-      href="<style-url>/images/favicons/favicon-32x32.png" >
+    <link rel="apple-touch-icon" sizes="57x57" href="<style-url>/images/favicons/apple-touch-icon-57x57.png">
+    <link rel="apple-touch-icon" sizes="114x114" href="<style-url>/images/favicons/apple-touch-icon-114x114.png">
+    <link rel="apple-touch-icon" sizes="72x72" href="<style-url>/images/favicons/apple-touch-icon-72x72.png">
+    <link rel="apple-touch-icon" sizes="144x144" href="<style-url>/images/favicons/apple-touch-icon-144x144.png">
+    <link rel="apple-touch-icon" sizes="60x60" href="<style-url>/images/favicons/apple-touch-icon-60x60.png">
+    <link rel="apple-touch-icon" sizes="120x120" href="<style-url>/images/favicons/apple-touch-icon-120x120.png">
+    <link rel="apple-touch-icon" sizes="76x76" href="<style-url>/images/favicons/apple-touch-icon-76x76.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="<style-url>/images/favicons/apple-touch-icon-152x152.png">
+    <link rel="apple-touch-icon" sizes="180x180" href="<style-url>/images/favicons/apple-touch-icon-180x180.png">
+    <link rel="icon" type="image/png" sizes="192x192" href="<style-url>/images/favicons/favicon-192x192.png">
+    <link rel="icon" type="image/png" sizes="160x160" href="<style-url>/images/favicons/favicon-160x160.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="<style-url>/images/favicons/favicon-96x96.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="<style-url>/images/favicons/favicon-16x16.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="<style-url>/images/favicons/favicon-32x32.png">
 
     <link rel="chrome-webstore-item" href="https://chrome.google.com/webstore/detail/lcgbonbkhleeeoibhgepeeeldjmgjhcm">
     <script src="lib/intl/dist/Intl.complete.js"></script>
     <script type="text/javascript">
-      ('Promise' in window && 'reject' in window.Promise) || document.write("<script src='lib/es6-promise/promise.min.js'><\/script>");
+        ('Promise' in window && 'reject' in window.Promise) || document.write("<script src='lib/es6-promise/promise.min.js'><\/script>");
```

Comment:
@4p00rv Could you please check with @ashkanx on this? If we are not supporting browsers which do not have native Promise support, we can handle it in Modernizr check and just recommend users to use the latest version of their browser.